### ISSUE TITLE
When there are no barlines

### ISSUE THREAD

**Issue by gregchapman-dev** [cuthbertLab/music21#1669]
In MusicXML, the absence of a <barline> element implies a “normal” single barline.  When this is imported into music21, there is no music21 Barline (measure.rightBarline and measure.leftBarline are both None).  Is this a bug in xmlToM21.py?  Or does music21 have the same implication that MusicXML does:  absence of Barline means normal barline?

**Comment by jacobtylerwalls** [comment]
Yeah, I think music21's assumptions are the same as musicxml. Absence of barline means normal barline. To represent "no barline", a Barline needs to be instantiated with a type of "none".

**Comment by jacobtylerwalls** [comment]
I'd happily merge a docs update to clarify!
